modules/monitoring-running-metrics-queries.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="running-metrics-queries_{context}"]
+= Running metrics queries
+
+You begin working with metrics by entering one or several Prometheus Query Language (PromQL) queries.
+
+.Procedure
+
+. Open the OpenShift Container Platform web console and navigate to *Monitoring* -> *Metrics*.
+
+. In the query field, enter your PromQL query.
+* To show all available metrics and PromQL functions, use the "Insert Metric at Cursor" button.
+. For multiple queries, use the "Add Query" button.
+. For deleting queries, click the action button for the query and select "Delete query".
+. For keeping but not running a query, click the "Disable query" button.
+. Once you finish creating queries, click the "Run Queries" button. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
```

Comment:
s/click the "Run Queries" button/click **Run Queries**/

Reply:
Implemented.